airflow/api/common/experimental/trigger_dag.py

```diff
@@ -20,7 +25,8 @@
 from airflow.utils.state import State
 
 
-def trigger_dag(dag_id, run_id=None, conf=None, execution_date=None):
+def trigger_dag(dag_id, run_id=None, conf=None, execution_date=None,
```

Comment:
Did you tested the operator locally to see if the test passes?

Reply:
Yes, its alive!! :D